### ISSUE TITLE
Complete outdoor audio tour with updated rover scripts

### ISSUE THREAD

**Issue by Moonwards** [moonwards1/Moonwards-Virtual-Moon#390]
The game has a Ground_Passenger_Pod docked at the N3 airlock. Previously the player has spawned inside that pod, and the N3 airlock doors were open. The AudioTourPaths begin and end at that spot because the intention was that after completing the indoor tour, the player would return to that pod and it would take it on the outdoor tour.

Ground_Passenger_Pods need to be carried around by an Athlete_Rover. Animations exist for the airlock doors, the door of the pod, and the rover. (The rover needs to be much better rigged and animated in order to be convincing when carrying the pod, see #291.) 

A script exists that gives the player a place to click inside the pod to activate the rover, and a sequence of animations that result. This is how the tour starts. When clicked
    - The doors of the airlock and the door of the pod close.
    - The rover backs away from the airlock, and raises into its travelling position
    -  It then proceeds to the first outdoor audio tour marker. The script did this with pathfinding, but doing it by having the rover follow a set path may give better results. The player didn't control the rover.
    -  For best visibility, the camera took a 3rd person perspective in which it could orbit around the rover to look at things
    -  The player can click on the markers the same as in the indoor tour
    -  After completing all the markers, the rover will return to the Airlock Bay and dock again at the same airlock. The animation sequence then goes in reverse, with the rover lowering to its docking position, easing forward to lock with the airlock door, and then the doors opening.

If control of the rover could be given to the player, possibly it would be a good option to guide them on the outdoor portion differently, allowing them to go to the markers on their own , or to leave it at some point to look at things on their own. However in that case, some way for them to understand how to get to the next marker would be needed.

**Comment by Zach777** [comment]
Do you need me to create larger AudioSpheres for the Rover to interact with?
 Should I make a wayfinder ala Crazy Taxi destination arrow?

**Comment by Moonwards** [comment]
@Zach777 I have been waiting on being able to run around with a rover to see how it feels. You are probably right about both things. 
They should be separate issue cards, should they not?

**Comment by Zach777** [comment]
Depends on whether we want a collab of people working off each others' work or not.
 Outdoor audio sphere work does fit in with the "complete audio tour" since they are literally a part of it.

**Comment by Reousa** [comment]
Weren't we going to have it be a more freely navigable environment instead?

**Comment by Moonwards** [comment]
A new issue sounds like a good idea. But having a wayfinder system seems like it could be a happy medium. For people interested in the tour, it could be a low key way to continue with it, at their own pace. Because the outdoors is so large, I think especially there it could be useful.

**Comment by Reousa** [comment]
I'm all for the navigation system yeah, we could have the rover automatically follow the paths, as well, in the future

**Comment by Reousa** [comment]
With the rover up and running, it makes sense that we close this card in favor of more detailed ones of what comes next @KimberleyHolder .
I'll let you close that one when we have the replacements

**Comment by Moonwards** [comment]
Let's see, i'm not sure of the logical way to split this up @Reousa. 
A card for 
* the undocking and docking animations
* the  waypoint navigation
* the interface where the player takes control of the rover
* the camera setup
and the rover controls and animation now exist. 
Does that make sense?

**Comment by Reousa** [comment]
Yep, I'll make the cards for them :thumbs_up:

**Comment by Reousa** [comment]
Should we close this now that we have independent issues for the remaining features?

**Comment by Reousa** [comment]
Closing this as we now have other issues that track these features independently.
